### PR TITLE
Azdatalake stg 78-82

### PR DESCRIPTION
**Pull request by tanyasethi-msft**
- [x] The purpose of this PR is explained in this or a referenced issue.
- [] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [x] Tests are included and/or updated for code changes.
- [x] Updates to module CHANGELOG.md are included.
- [x] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go

### CODE REVIEW

File: sdk/storage/azdatalake/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## 1.0.1 (Unreleased)
 
 ### Features Added
+* Encryption Scope For SAS
+* CPK for Datalake
+* Create SubDirectory Client
```
Comment (souravgupta-msft) [inline sdk/storage/azdatalake/CHANGELOG.md:8]: Add a changelog entry indicating service version upgrade.

Comment (tanyasethi-msft) [reply sdk/storage/azdatalake/CHANGELOG.md:8]: Added

**Review by souravgupta-msft — APPROVED** [review]
Looks good. This PR contains changes from already approved PRs.

**Pull request merged**